gubernator/github/classifier.py

```diff
@@ -296,8 +296,6 @@ def get_reviewers(events):
                 reviewers.add(body['requested_reviewer']['login'])
             elif action == 'review_request_removed':
                 reviewers -= {body['requested_reviewer']['login']}
-        elif event == 'pull_request_review' and action == 'submitted':
```

Comment:
No, you see the same "submitted" event even for someone else just commenting on a review thread.